content/es/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,164 @@
+* No dictamina las soluciones de registros, monitoreo o alerta que se deben usar. Hay algunas integraciones que se ofrecen como prueba de concepto, y existen mecanismos para recolectar y exportar métricas.
+* No provee ni obliga a usar un sistema o lenguaje de configuración (como [jsonnet](https://github.com/google/jsonnet)) sino que ofrece una API declarativa que puede ser usada con cualquier forma de especificación declarativa
+* No provee ni adopta un sistema exhaustivo de mantenimiento, administración o corrección automática de errores
+
+Además, Kubernetes no es un mero *sistema de orquestación*. De hecho, Kubernetes elimina la necesidad de orquestar. *Orquestación* se define como la ejecución de un flujo de trabajo definido: haz A, luego B y entonces C. Kubernetes está compuesto de un conjunto de procesos de control independientes y combinables entre si que llevan el estado actual hacia el estado deseado. No debería importar demasiado como llegar de A a C. No se requiere control centralizado y, como resultado, el sistema es más fácil de usar, más poderoso, robusto, resiliente y extensible.
+
+## ¿Por qué usar contenedores?
+
+¿Te preguntas las razones para usar contenedores?
+
+![Why Containers?](/images/docs/why_containers.svg)
+
+La *Manera Antigua* de desplegar aplicaciones era instalarlas en un
+servidor usando el administrador de paquetes del sistema operativo.
+La desventaja era que los ejecutables, la configuración, las librerías
+y el ciclo de vida de todos estos componentes se entretejían unos a
+otros. Podíamos construir imágenes de máquina virtual inmutables para
+tener rollouts y rollbacks predecibles, pero las máquinas virtuales
+son pesadas y poco portables.
+
+La *Manera Nueva* es desplegar contenedores basados en virtualización
+a nivel del sistema operativo, en vez del hardware. Estos contenedores
+están aislados entre ellos y con el servidor anfitrión: tienen sus propios
+sistemas de archivos, no ven los procesos de los demás y el uso de recursos
+puede ser limitado. Son más fáciles de construir que una máquina virtual, y
+porque no están acoplados a la infraestructura y sistema de archivos del
+anfitrión, pueden llevarse entre nubes y distribuciones de sistema operativo.
+
+Ya que los contenedores son pequeños y rápidos, una aplicación puede ser
+empaquetada en una imagen de contenedor. Esta relación uno a uno entre
+aplicación e imagen nos abre un abanico de beneficios para usar contenedores.
+Con contenedores, podemos crear imágenes inmutables al momento de la compilación
+en vez del despliegue ya que las aplicaciones no necesitan componerse junto al
+resto de la pila ni atarse al ambiente de infraestructura de producción. Generar
```

Comment:
```suggestion
resto del _stack_ ni atarse al entorno de infraestructura de producción. Generar
```

En este caso, se refiere al stack formado por el todas las aplicaciones relacionadas. No se suele usar la palabra pila en este contexto, al menos en España. Cómo tu veas!

Reply:
Si que me encuentro con "pila" con cierta frecuencia, pero para la mayoría de términos de uso común siempre soy partidario de dejarlos sin traducir y en itálicas como sugieres